core/src/main/scala/kafka/coordinator/group/MemberMetadata.scala

```diff
@@ -66,7 +66,7 @@ private[group] class MemberMetadata(var memberId: String,
   var assignment: Array[Byte] = Array.empty[Byte]
   var awaitingJoinCallback: JoinGroupResult => Unit = null
   var awaitingSyncCallback: SyncGroupResult => Unit = null
-  var latestHeartbeat: Long = -1
```

Comment:
Sad to see this go since it was useful when things went wrong. But since it is not used anymore, I guess it must go.

Reply:
I think its current usage was a bit error-prone. However, if it's useful for debugging, we can find a way to keep it. It did help us understand this bug after all. Perhaps we could have a separate field for the last time a request was received?